tests/unittests/test_shared_memory_map.py

```diff
@@ -20,7 +20,6 @@
 @skipIf(sys.platform == 'darwin', 'MacOS M1 machines do not correctly test the'
                                   'shared memory filesystems and thus skipping'
                                   ' these tests for the time being')
-@pytest.mark.flaky(reruns=3)
```

Comment:
Why move the flaky from class level to specific tests?

Reply:
pytest.mark.flaky does not work for class level.